opencensus/common/monitored_resource/monitored_resource.py

```diff
@@ -86,4 +86,4 @@ def get_instance():
 
     if not resources:
         return None
-    return resource.merge_resources(resources)
+    return resource.merge_resources(resources[0], *resources[1:])
```

Comment:
There's not really a beautiful way to do this here since any of the resources could be null.